tests/e2e/specs/dashboard-samples/RecomendedExtentions.spec.ts

```diff
@@ -0,0 +1,124 @@
+            await driverHelper.waitVisibility(webCheCodeLocators.Editor.inputArea);
+            Logger.debug(`editor.getText(): get recommended extensions as text from editor, delete comments and parse to object.`);
+            recommendedExtensions = JSON.parse((await editor.getText()).replace(/\/\*[\s\S]*?\*\/|(?<=[^:])\/\/.*|^\/\/.*/g, '').trim());
+            Logger.debug(`recommendedExtensions.recommendations: Get recommendations clear names using map().`);
+            recommendedExtensions.recommendations = recommendedExtensions.recommendations.map((r: string) => r.substring(r.indexOf('.') + 1, r.length));
+        });
+
+        test(`Check if recommended extensions were installed`, async function (): Promise<void> {
+            Logger.debug(`ActivityBar().getViewControl('Extensions'))?.openView(): open Extensions view.`);
+            const extensionsView: SideBarView | undefined = await (await new ActivityBar().getViewControl('Extensions'))?.openView();
+            Logger.debug(`extensionsView?.getContent().getSections(): get current section.`);
+            const [extensionSection]: ExtensionsViewSection[] = await extensionsView?.getContent().getSections() as ExtensionsViewSection[];
+            Logger.debug(`Time for extensions installation TimeoutConstants.TS_COMMON_PLUGIN_TEST_TIMEOUT=${TimeoutConstants.TS_COMMON_PLUGIN_TEST_TIMEOUT}`);
+            await driverHelper.wait(TimeoutConstants.TS_COMMON_PLUGIN_TEST_TIMEOUT);
+            Logger.debug(`Check if recommendedExtensions.recommendations are installed: ${recommendedExtensions.recommendations}.`);
+            for (const extension of recommendedExtensions.recommendations) {
+                Logger.debug(`extensionSection.findItem(${extension}).`);
+                await extensionSection.findItem(extension);
+                Logger.debug(`extensionsView?.getContent().getSections(): switch to marketplace section.`);
+                const [marketplaceSection]: ExtensionsViewSection[] = await extensionsView?.getContent().getSections() as ExtensionsViewSection[];
+                await driverHelper.waitVisibility(webCheCodeLocators.ExtensionsViewSection.items, TimeoutConstants.TS_COMMON_PLUGIN_TEST_TIMEOUT);
+                Logger.debug(`marketplaceSection.getVisibleItems(): get first item.`);
+                const [firstFoundItem]: ExtensionsViewItem[] = await marketplaceSection.getVisibleItems();
+                Logger.debug(`firstFoundItem?.isInstalled()`);
+                const isInstalled: boolean = await firstFoundItem?.isInstalled() as boolean;
+                Logger.debug(`firstFoundItem?.isInstalled(): ${isInstalled}.`);
+                expect(isInstalled).eqls(true);
+            }
+        });
+
+            test('Stopping and deleting the workspace', async function (): Promise<void> {
+                await workspaceHandlingTests.stopAndRemoveWorkspace(WorkspaceHandlingTests.getWorkspaceName());
+            });
+        }
+
+    loginTests.logoutFromChe();
```

Comment:
It's interesting to know what is the propose of this step?

Reply:
to execute a few test spec files one by one in one run

Reply:
Got it.

Not sure if it will it help when there are several tests run in parallel ;-)